includes/class-newspack-newsletters.php

```diff
@@ -1096,7 +1266,7 @@ public static function activation_nag_dismissal_script() {
 				'ajaxurl' => get_admin_url() . 'admin-ajax.php',
 			]
 		);
-		wp_enqueue_script( $script );
+		wp_2ue_script( $script );
```

Comment:
I don't think this change was on purpose? As far as I can tell, this function does not exist, and this will cause a fatal error when the plugin is activated if Newspack Newsletters hasn't been set up.

Reply:
This was by mistake changed seems to be. Let me correct it. I have noticed this yesterday and updated commit seems to be it is not updated. Let me update it today